libs/designsystem/src/lib/components/modal/services/modal-navigation.service.ts

```diff
@@ -212,17 +212,19 @@ export class ModalNavigationService {
     return { activated$: EMPTY, deactivated$: EMPTY };
   }
 
-  async navigateToModal(path: string | string[]): Promise<boolean> {
+  async navigateToModal(path: string | string[], queryParams?: Params | null): Promise<boolean> {
```

Comment:
Why the null type? 🤔
```suggestion
  async navigateToModal(path: string | string[], queryParams?: Params): Promise<boolean> {
```